Documentation/Usage.md

```diff
@@ -0,0 +1,84 @@
+# AEP Assurance Android Public API
+
+---
+
+## extensionVersion
+
+Returns the current version of the AEP Assurance extension.
+
+**Signature**
+
+```java
+public static String extensionVersion()
+```
+
+**Usage**
+
+#### Java
+
+```java
+final String extensionVersion = Assurance.extensionVersion();
+```
+
+#### Kotlin
+
+```kotlin
+val extensionVersion: String = Assurance.extensionVersion()
+```
+
+---
+
+## startSession
+
+The `startSession` API can be called to begin a AEP Assurance session. When called, the Mobile SDK displays a PIN authentication overlay to begin a session.
+
+**Signature**
+
+```java
+public static void startSession(final String url)
+```
+
+**Usage**
+
+#### Java
+
+```java
+final String url = "<assurance_session_url>"
+Assurance.startSession(url);
+```
+
+#### Kotlin
+
+```kotlin
+val url: String = "<assurance_session_url>"
+Assurance.startSession(url)
+```
+
+---
+
+## EXTENSION
+
+Represents a reference to `AssuranceExtension.class` that can be used to register with `MobileCore` via its `registerExtensions` api.
+
+**Signature**
+
+```java
+public static final Class<? extends Extension> EXTENSION = AssuranceExtension.class;
+```
+
+**Usage**
+
+#### Java
+
+```java
+MobileCore.registerExtensions(Arrays.asList(Assurance.EXTENSION), new AdobeCallback<Object>() {
```

Comment:
Can you replace `Arrays.asList(Assurance.EXTENSION)` to `Arrays.asList(Assurance.EXTENSION, ...)`